.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -12,11 +12,11 @@ with @pairperson1
 ## Type of change
 Please delete options that are not relevant.
 
-- [ ] Bug fix (non-breaking change which fixes an issue)
-- [ ] New feature (non-breaking change which adds functionality)
-- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-- [ ] This change requires a documentation update
-- [ ] This change requires updated or new tests
+- Bug fix (non-breaking change which fixes an issue)
+- New feature (non-breaking change which adds functionality)
+- Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- This change requires a documentation update
+- This change requires updated or new tests
```

Comment:
this is just so we don't have things that look like tasks on our PRs, but it's unrelated to these other changes